### PR TITLE
docs(flypad-translations): add localazy logo and partnership information

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Added logo and associated sponor referral link to our flyPadOS 3 localization documentation.

Required markdown extension `md_in_html` to be added to support image centering in mkdocs

### Location
- docs/dev-corner/dev-guide/specific/flypad-translations.md
- mkdocs.yml

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

File: docs/dev-corner/dev-guide/specific/flypad-translations.md [hunk 1]
```diff
@@ -2,4 +2,13 @@
 
 This page is pulled externally from the A32NX repository and describes the processes involved with adding features to the EFB and sourcing translations.
 
+We're thrilled to be partnered with Localazy, who offers top-notch software localization services bringing multiple language support to our EFB enhancing the user experience.
```
Comment (Sleinmaster) [inline docs/dev-corner/dev-guide/specific/flypad-translations.md]: Unlike a marketing announcement the docs should be written in a timeless manner in my opinion. 
I'll make a suggestion later tonight also with some grammar corrections.

Comment (Valastiri) [reply docs/dev-corner/dev-guide/specific/flypad-translations.md]: Excellent thanks! this is meant to be a draft that's my bad -- let me know here or Discord. Will set PR accordingly.

Comment (Sleinmaster) [inline docs/dev-corner/dev-guide/specific/flypad-translations.md]:
```suggestion
We have partnered with Localazy, who offer top-notch software localization services, bringing multi-language support to our EFB, enhancing the user experience.
```

Comment (Valastiri) [reply docs/dev-corner/dev-guide/specific/flypad-translations.md]: Would it be better to keep it "partnered"? Otherwise this would be good.

Comment (Sleinmaster) [reply docs/dev-corner/dev-guide/specific/flypad-translations.md]: "We have partnered" would be even better.

Comment (Valastiri) [reply docs/dev-corner/dev-guide/specific/flypad-translations.md]: Done deal! Thanks Slein

**Pull request merged**